FIDO2_authenticate_credential_req.md

```diff
@@ -0,0 +1,27 @@
+## FIDO2 Authenticate Credential Request
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
ditto